diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/DiktatRuleSetProvider.kt

```diff
@@ -81,6 +94,12 @@ class DiktatRuleSetProvider(private val diktatConfigFile: String = "diktat-analy
         )
     }
 
+    private fun validate(config: RulesConfig) =
+        require(config.name == DIKTAT_COMMON || config.name in Warnings.names) {
```

Comment:
it will do procExit(1) and stop diktat (diktat should be stopped in this case I guess)
We do not handle this exception somehow accidentally?

Reply:
I think, when running via KtLint, this exception will be thrown all the way to ktlint, when it's loading discovered service providers. In `diktat-maven-plugin` it's not handled and will cause `MojoExecutionException`.
CLI exception will look like this:
```
Exception in thread "main" java.lang.IllegalArgumentException: Warning name <CLASS_NAME_INCORRECT__> in configuration file is invalid, did you mean <CLASS_NAME_INCORRECT>?
	at org.cqfn.diktat.ruleset.rules.DiktatRuleSetProvider.validate(DiktatRuleSetProvider.kt:101)
	at org.cqfn.diktat.ruleset.rules.DiktatRuleSetProvider.get(DiktatRuleSetProvider.kt:44)
	at com.pinterest.ktlint.KtlintCommandLine.loadRulesets(Main.kt:550)
	at com.pinterest.ktlint.KtlintCommandLine.run(Main.kt:222)
	at com.pinterest.ktlint.Main.main(Main.kt:62)
```